.golangci.yml

```
@@ -1,14 +1,8 @@
 ---
-output:
-  # print lines of code with issue, default is true
-  print-issued-lines: false
-
-# all available settings of specific linters
 linters-settings:
-  govet:
-    enable-all: true
-    disable:
-      - fieldalignment
+  stylecheck:
+    # STxxxx checks in https://staticcheck.io/docs/configuration/options/#checks
+    checks: ["all", "-ST1000"]
```

nit: We seem to use yaml list syntax for other lists:

```suggestion
    checks:
      - all
      - -ST1000
```